inc/template-tags.php

```diff
@@ -181,9 +186,8 @@ function twentysixteen_post_thumbnail() {
 function twentysixteen_excerpt_more( $more ) {
 	$link = sprintf( '<a href="%1$s" class="more-link">%2$s</a>',
 		esc_url( get_permalink( get_the_ID() ) ),
-		/* translators: %s: Name of current post */
-		sprintf( __( 'Continue reading %s', 'twentysixteen' ), '<span class="screen-reader-text">' . esc_html( get_the_title( get_the_ID() ) ) . '</span>' )
-		);
+		sprintf( _x( 'Continue reading %s', 'Name of current post', 'twentysixteen' ), '<span class="screen-reader-text">' . esc_html( get_the_title( get_the_ID() ) ) . '</span>' )
```

Comment:
The way with the translation comment works fine. I don't think we need to change it to `_x()`